docs/UserGuide.md

```diff
@@ -39,7 +39,8 @@ title: User Guide
 
 3. Copy the file to the folder you want to use as the _home folder_ for your **CinternS** app.
 
-4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. When the app is started for the first time, sample data is included to help you get started.<br>
+4. Open a command window in the directory containing the `CinternS.jar` file and launch it with the command `java -jar CinternS.jar`. <br><br>
```

Comment:
Maybe we can keep double clicking as the default option, then say that if that does not work then try the command instead?